### PR TITLE
(maint) Prepare for 2.1.1 release

### PR DESCRIPTION
**Pull request by sarameisburger**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 2.1.1 - 2019-08-21
+This is a maintenance release.
+
+Bugfix:
+  * (SERVER-2602) Do not manage egd. For FIPS enabled builds explicitly set this to `/dev/random` via the `JAVA_ARGS` variable. FOSS users who would like to manage their egd source should use `JAVA_ARGS` as well.
```
Comment (sarameisburger) [inline CHANGELOG.md]: @justinstoller is this a backwards-incompatible change?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+  * Fix deprecation warning to suggest using `Pkg::Util::Git.fail_on_dirty_source`.
```
Comment (underscorgan) [inline CHANGELOG.md]: This is worded a little strangely. Maybe something more like 'Stop calling deprecated method `Pkg::Util::Version.fail_on_dirty_source`' ?

File: CHANGELOG.md (hunk 1 above)
Comment (justinstoller) [reply CHANGELOG.md]: AFAICT, this change doesn't have an effect when running on Java >= 8.

File: CHANGELOG.md (hunk 2 above)
Comment (sarameisburger) [reply CHANGELOG.md]: sounds good. i wasn't sure the best way to word it

Comment (sarameisburger) [reply CHANGELOG.md]: alrighty i updated it

File: CHANGELOG.md (hunk 1 above)
Comment (sarameisburger) [reply CHANGELOG.md]: ahh gotcha. and is this description accurate? there were references to `dev/random` and `dev/urandom` in the PR

Comment (mwaggett) [inline CHANGELOG.md]:
```suggestion
  * (SERVER-2602) Do not manage egd. For FIPS-enabled builds, explicitly set this to `/dev/random` via the `JAVA_ARGS` variable. FOSS users who would like to manage their egd source should use `JAVA_ARGS` as well.
```

Comment (justinstoller) [reply CHANGELOG.md]: The middle sentence is very PE FIPS build specific and could probably be dropped. Something like: "Do not hard code egd in included subcommand files. Users that would like to manage egd source may do so via JAVA_ARGS (or their java.security file" I think contains the info that a FOSS user/consumer would want to know. (sorry, I should have written a better commit message)

Comment (sarameisburger) [reply CHANGELOG.md]: alrighty, updated!

**Pull request merged**